fastlane/actions/project_name.rb

```diff
@@ -0,0 +1,71 @@
+require 'xcodeproj'
+
+module Fastlane
+  module Actions
+    class ProjectNameAction < Action
+
+      PROJECT_EXTENSION = ".xcodeproj".freeze
+
+      # This script returns the application name
+      # if there is a scheme that matches the project name.
+      # Otherwise, it assumes the project name should be specified 
+      # by the user and fails.
+
+      def self.run(params)
+        default_project_name
```

Comment:
params don't matter? If the user passes the project name?

Reply:
Yeah, there was a bug. Fixed now 👍